packages/rx_bloc_cli/example/README.md

```diff
@@ -380,6 +382,57 @@ All additional info about package and better explanation how to implement you ca
 
 
 
+## Dev Menu
```

Comment:
Please do not overload the main README.md file with too many details. One of the options is to keep the overview of the DevMenu in the readme file and to move the detailed explanations to a separate one (docs/DEVMENU.md)